File: abm-personas-fe/src/types/persona.ts
import Base from "./Base";

export default class Persona extends Base{
  email: string;
  firstName: string;
  lastName: string;

  constructor (
    id: number = 0,
    email: string = "",
    firstName: string= "",
    lastName: string= "",
  ){
    super(id);
    this.email= email;
    this.firstName = firstName;
    this.lastName = lastName;
  }
}

File: PersonasBack/src/main/java/com/example/instrumentosback/entities/Persona.java
package com.example.instrumentosback.entities;


import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Table(name = "personas")
@NoArgsConstructor
@Getter
@Setter
@AllArgsConstructor
public class Persona extends Base {

    @Column(name="email")
    private String email;

    @Column(name="firstName")
    private String firstName;

    @Column(name="lastName")
    private String lastName;

}
